### ISSUE TITLE
[Doc] Keyword for tutorials

### ISSUE THREAD

**Issue by J-shang** [microsoft/nni#4681]
[Start and Manage a New Experiment](https://nni.readthedocs.io/en/latest/tutorials/nni_experiment.html)  Remove
[NNI HPO Quickstart with PyTorch](https://nni.readthedocs.io/en/latest/tutorials/hpo_quickstart_pytorch/main.html)
[NNI HPO Quickstart with TensorFlow](https://nni.readthedocs.io/en/latest/tutorials/hpo_quickstart_tensorflow/main.html)
[Hello, NAS!](https://nni.readthedocs.io/en/latest/tutorials/hello_nas.html)
[Use NAS Benchmarks as Datasets](https://nni.readthedocs.io/en/latest/tutorials/nasbench_as_dataset.html)
[Pruning Quickstart](https://nni.readthedocs.io/en/latest/tutorials/pruning_quick_start_mnist.html)
[Speed Up Model with Mask](https://nni.readthedocs.io/en/latest/tutorials/pruning_speed_up.html)
[Quantization Quickstart](https://nni.readthedocs.io/en/latest/tutorials/quantization_quick_start_mnist.html)
[Speed Up Model with Calibration Config](https://nni.readthedocs.io/en/latest/tutorials/quantization_speed_up.html)

**Comment by matluster** [comment]
I think we can put the keywords in tutorials tags.

Hello NAS!: NAS, Experiment, Multi-trial
NAS benchmarks as datasets: NAS, Benchmark

**Comment by cruiseliu** [comment]
## HPO ##

Explanation: HPO means hyperparameter optimization, which can be understood as "find best values" or "change the value to a better one". The `{ a ⇒ ɑ }` logo was for HPO.

Tutorial tags:

* Quickstart with PyTorch
  1. HPO
  2. PyTorch: suggest to use [PyTorch logo](https://avatars.githubusercontent.com/u/21003710)
  3. [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist): dataset of clothes
  4. Quickstart (optional)
* Quickstart with TensorFlow
  1. HPO
  2. TensorFlow: suggest to use [TensorFlow logo](https://avatars.githubusercontent.com/u/15658638)
  3. [MNIST](https://en.wikipedia.org/wiki/MNIST_database): dataset of handwritten digits.
  4. Quickstart (optional)

Planned future tutorials: (for reference only, no need to design now)

* Quickstart with sklearn:
  1. HPO
  2. [Scikit-learn](https://avatars.githubusercontent.com)
  3. Classification
  4. Quickstart
* Hello HPO in pure Python
  1. HPO
  2. Pure Python
  3. Branin function
  4. Hello world
* Remote training service
  1. HPO
  2. SSH
  3. [On-premise](https://cn.bing.com/images/search?q=on-premise)
* Distributed trial
  1. HPO
  2. [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) cloud
  3. Distributed

**Comment by J-shang** [comment]
- Pruning Quickstart: Compression, Pruning
- Speed Up Model with Mask: Compression, Pruning, Speedup
- Quantization Quickstart: Compression, Quantization
- Speed Up Model with Calibration Config:  Compression, Quantization, Speedup

**Comment by scarlett2018** [comment]
@J-shang - shall we close this issue? are we still working on any of these items?

**Comment by J-shang** [comment]
> @J-shang - shall we close this issue? are we still working on any of these items?

yes, we can close this
